adapter/ipListChecker/adapter.go

```diff
@@ -15,74 +15,93 @@
 package ipListChecker
 
 import (
-	"errors"
+	"fmt"
+	"net/http"
 	"net/url"
 	"time"
 
-	"istio.io/mixer/pkg/adapter"
+	"github.com/golang/protobuf/proto"
+
+	"istio.io/mixer/pkg/aspect/listChecker"
+	"istio.io/mixer/pkg/aspectsupport"
+)
+
+const (
+	// ImplName is the canonical name of this implementation
+	ImplName = "istio/IPListChecker"
```

Comment:
Why this name does not match package name? It seems confusing to use slightly different name for the same thing.